### PR TITLE
fix(FixedFooterLayout): fix warnings in SSR

### PR DESCRIPTION
**Pull request by marcoskolodny**

### CODE REVIEW

**Review by Marcosld — APPROVED** [review]

File: src/fixed-footer-layout.tsx
Comment (atabel) [inline src/fixed-footer-layout.tsx:1]: I miss a SSR test detecting this

File: src/fixed-footer-layout.tsx [hunk 1]
```diff
@@ -106,17 +105,17 @@ const FixedFooterLayout: React.FC<Props> = ({
         };
     }, [hasContentEnoughVSpace, platformOverrides]);
 
-    const isContentWithScroll = isClientSide() && hasScroll(getScrollableParentElement(containerRef.current));
-
-    const isFixedFooter = hasContentEnoughVSpace || !isContentWithScroll;
+    const isFixedFooter = hasContentEnoughVSpace;
```
Comment (marcoskolodny) [inline src/fixed-footer-layout.tsx:108]: After debugging in different devices (mobile and web) and testing several scenarios with @brtbrt, we concluded that it's useless to check for the condition `!isContentWithScroll`. This was old logic, and it makes no sense to use it in the current code.

**Review by atabel — APPROVED** [review]

File: src/fixed-footer-layout.tsx (hunk 1 above)
Comment (brtbrt) [reply src/fixed-footer-layout.tsx:108]: specifically, it come from the old JSS styling 👍

**Review by brtbrt — APPROVED** [review]

**Pull request merged**